### PR TITLE
Tweak: Added more sections on System info [ED-8741]

### PR DESCRIPTION
**Pull request by KingYes**

### CODE REVIEW

File: modules/usage/module.php [hunk 1]
```diff
@@ -582,6 +582,35 @@ private function save_document_usage( Document $document ) {
 		}
 	}
 
+	public static function get_settings_usage() {
+		$usage = [];
+
+		$settings_tab = Plugin::$instance->settings->get_tabs();
+		$settings = array_merge( $settings_tab[ \Elementor\Settings::TAB_GENERAL ]['sections'], $settings_tab[ \Elementor\Settings::TAB_ADVANCED ]['sections'] );
```
Comment (StyleShit) [inline modules/usage/module.php:589]:
```suggestion
		$settings = array_merge( 
			$settings_tab[ Settings::TAB_GENERAL ]['sections'], 
			$settings_tab[ Settings::TAB_ADVANCED ]['sections']
		);
```

File: modules/usage/module.php [hunk 2]
```diff
@@ -582,6 +582,35 @@ private function save_document_usage( Document $document ) {
...
+					continue;
+				}
+
+				$setting_value = get_option( 'elementor_' . $field_name );
+
+				if ( empty( $setting_value ) ) {
+					continue;
+				}
+
+				if ( ! empty( $field_data['field_args']['std'] ) && $setting_value === $field_data['field_args']['std'] ) {
```
Comment (StyleShit) [inline modules/usage/module.php:603]: Maybe extract to `$is_default_value`?

File: modules/usage/settings-reporter.php [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+		return [
+			'settings' => '',
+		];
+	}
+
+	public function get_settings() : array {
+		$usage_settings_text = '';
+
+		$settings = Module::get_settings_usage();
+		foreach ( $settings as $setting_name => $setting_value ) {
```
Comment (StyleShit) [inline modules/usage/settings-reporter.php:26]:
```suggestion
		$usage_settings_text = '';
		$settings = Module::get_settings_usage();
		
		foreach ( $settings as $setting_name => $setting_value ) {
```

File: modules/usage/settings-reporter.php [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+		return [
+			'value' => $usage_settings_text,
+		];
+	}
+
+	public function get_raw_settings() : array {
+		$usage_settings = PHP_EOL;
+
+		$settings = Module::get_settings_usage();
```
Comment (StyleShit) [inline modules/usage/settings-reporter.php:40]:
```suggestion
		$usage_settings = PHP_EOL;
		$settings = Module::get_settings_usage();
		
```

File: modules/usage/settings-reporter.php [hunk 5]
```diff
@@ -0,0 +1,51 @@
+<?php
+namespace Elementor\Modules\Usage;
+
+use Elementor\Modules\System_Info\Reporters\Base as Base_Reporter;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Settings_Reporter extends Base_Reporter {
```
Comment (StyleShit) [inline modules/usage/settings-reporter.php:10]: tests?

**Review by StyleShit — CHANGES_REQUESTED** [review]

**Review by Nevoss — APPROVED** [review]

**Review by Nevoss — CHANGES_REQUESTED** [review]
Without test? and there is a lot of workflow that failed

File: tests/phpunit/elementor/modules/usage/test-module.php [hunk 6]
```diff
@@ -360,4 +361,22 @@ private function ensure_dynamic_tags() {
...
 	}
+
+	public function test_get_settings_usage() {
+		// Arrange.
+		update_option( 'elementor_disable_color_schemes', 'no' );
+		update_option( 'elementor_css_print_method', 'external' );
+		update_option( 'elementor_editor_break_lines', '1' );
+
+		update_option( Settings::UPDATE_TIME_FIELD, time() );
+
```
Comment (Nevoss) [inline tests/phpunit/elementor/modules/usage/test-module.php:372]:
```suggestion
		update_option( 'elementor_disable_color_schemes', 'no' );
		update_option( 'elementor_css_print_method', 'external' ); // Default value should not exist in the result
		update_option( 'elementor_editor_break_lines', '1' );
		
		update_option( Settings::UPDATE_TIME_FIELD, time() ); // Not an Elementor option should not exist in the result

```

**Review by Nevoss — CHANGES_REQUESTED** [review]

File: tests/phpunit/elementor/modules/usage/test-module.php [hunk 7]
```diff
@@ -360,4 +361,22 @@ private function ensure_dynamic_tags() {
...
+		// Act
+		$settings_usage = Module::get_settings_usage();
+
+		// Assert
+		$this->assertEquals( 'no', $settings_usage['disable_color_schemes'] );
+		$this->assertArrayNotHasKey( Settings::UPDATE_TIME_FIELD, $settings_usage );
+		$this->assertArrayNotHasKey( 'css_print_method', $settings_usage );
+		$this->assertEquals( '1', $settings_usage['editor_break_lines'] );
```
Comment (Nevoss) [inline tests/phpunit/elementor/modules/usage/test-module.php:380]:
```suggestion
		$this->assertEquals( 'no', $settings_usage['disable_color_schemes'] );
		$this->assertEquals( '1', $settings_usage['editor_break_lines'] );

		$this->assertArrayNotHasKey( 'css_print_method', $settings_usage );
		$this->assertArrayNotHasKey( Settings::UPDATE_TIME_FIELD, $settings_usage );
```

**Review by Nevoss — CHANGES_REQUESTED** [review]

**Review by Nevoss — APPROVED** [review]

**Pull request merged**